web/src/main/java/com/epam/esm/web/advice/CustomAdvice.java

```diff
@@ -0,0 +1,88 @@
+package com.epam.esm.web.advice;
+
+import com.epam.esm.service.exeption.IllegalRequestParameterException;
+import com.epam.esm.service.exeption.IllegalRequestSortParameterException;
+import com.epam.esm.service.exeption.RecourseExistException;
+import com.epam.esm.service.exeption.RecourseNotExistException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.util.Locale;
+
+@ControllerAdvice
+public class CustomAdvice extends ResponseEntityExceptionHandler {
+    @ExceptionHandler(RecourseExistException.class)
+    public ResponseEntity<ErrorResponse> handleRecourseExistException(RecourseExistException e) {
+        ErrorResponse response = new ErrorResponse();
+        response.setMessage(e.getMessage());
+        response.setLocalDateTime(LocalDateTime.now());
+        String str=CustomErrorCode.RECOURSE_EXIST.getStatusCode();
+        response.setStatusCode(CustomErrorCode.RECOURSE_EXIST.getStatusCode());
+        return new ResponseEntity<>(response, HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
it's not an `INTERNAL_SERVICE_ERROR` case

Reply:
done